server/src/main/java/org/candlepin/audit/EventSinkImpl.java

```diff
@@ -111,7 +115,10 @@ public void queueEvent(Event event) {
             return;
         }
 
-        modeManager.throwRestEasyExceptionIfInSuspendMode();
+        if (this.modeManager.getCurrentMode() != Mode.NORMAL) {
+            throw new IllegalStateException("Candlepin is in suspend mode");
```

Comment:
We shouldn't be throwing REST exceptions from deep within a given subsystem. If we *really* need that exception, it should be handled by the appropriate endpoint and then converted to the proper REST exception.

In this case, since we have suspend mode handling already, the only time we should hit this is if something is misbehaving and allowing event-generating code to run while suspended.